### PR TITLE
Patool

### PR DESCRIPTION
**Pull request by TheChymera**
New ebuild for [patool](https://pypi.python.org/pypi/patool/) - a popular Python archive management library.

### CODE REVIEW

**Comment by monsieurp** [comment]
@gentoo/python

File: app-arch/patool/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <!-- maintainer-needed -->
```
Comment (monsieurp) [inline app-arch/patool/metadata.xml]: Why would we add a new package to the tree if there's no maintainer to look after it? Doesn't make much sense, does it?

**Comment by monsieurp** [comment]
Thanks for your PR. Two things I'd like to point out:
* Could you give a little bit more background as to what this package is useful for and what sort of problem it is meant to be solving ? The sentence "New package" speaks for itself but doesn't give much information.
* When a contributor wants to add a new package to the Portage tree, we usually expect him to look after it. Please have a read through the proxy maintainer page https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers and feel free to update the metadata.xml file. Also, don't forget to file a bug so that your info are stored in Bugzilla. Cheers.

**Comment by TheChymera** [comment]
* Patool is a simple python package designed as a one-stop-shop for dealing with archives. It provides built-in functionality for extracting, creating, comparing, and repacking the most popular archive formats. It is useful for simplifying archive-related workflows. I use it for a few of my low-key projects, and it is also required for at least [one sizable data management project](http://datalad.org/). 

* Maintainership request [on Bugzilla](https://bugs.gentoo.org/show_bug.cgi?id=587792)

**Comment by SoapGentoo** [comment]
Dear Christian,
could you squash your commits into one? Then I can merge it.

**Comment by TheChymera** [comment]
@SoapGentoo seems t have been merged by the bot already?

**Comment by SoapGentoo** [comment]
@TheChymera Yes, I've merged it in the meantime. Please always squash your commits into one logical change. Makes for easier reverting 😄

**Comment by TheChymera** [comment]
Many thanks!

**Pull request closed without merging**